### PR TITLE
Add missed isSimilar examples in traits.md.

### PR DESCRIPTION
**Pull request by eryshev**
It would be better for newcomers to see some `isSimilar` examples in this doc.

### CODE REVIEW

File: tutorials/tour/traits.md [hunk 1]
```diff
@@ -34,16 +34,20 @@ object TraitsTest extends App {
   val p1 = new Point(2, 3)
   val p2 = new Point(2, 4)
   val p3 = new Point(3, 3)
-  println(p1.isNotSimilar(p2))
-  println(p1.isNotSimilar(p3))
+  val p4 = new Point(2, 3)
+  println(p1.isSimilar(p2))
+  println(p1.isSimilar(p3))
+  // Point's isNotSimilar is defined in Similarity
   println(p1.isNotSimilar(2))
```
Comment (SethTisue) [inline tutorials/tour/traits.md]: is `2` supposed to be `p2`?

Comment (eryshev) [inline tutorials/tour/traits.md]: No, it supposed to illustrate that similarity test fails on `2.isInstanceOf[Point]` test !

Comment (SethTisue) [inline tutorials/tour/traits.md]: doh, right

**Comment by SethTisue** [comment]
thanks!

**Pull request merged**